test/browser/browser.sh

```diff
@@ -46,6 +46,17 @@ if grep -q 'ID=.*rhel' /etc/os-release; then
     dnf install -y kpatch kpatch-dnf
 fi
 
+# HACK: RHEL has these bundled in cockpit-system, but Fedora doesn't; Provides: break in CentOS/RHEL 10
+# due to https://issues.redhat.com/browse/TFT-2564
+if ! grep -q platform:el /etc/os-release; then
```

Comment:
Should this also not be el10?

Reply:
Nevermind, didn't see the changes below, but then we can drop the `platform:el` grep no? As we have to do it for every image?

Reply:
This is quite deliberate. *All* RHEL versions bundle the modules in cockpit-system, so we must not try to `dnf install cockpit-kdump` on RHEL 8/9 either.

Well, "must", it'll be idempotent and redundant, but conceptually it's cleaner that way and matches the comment.